### ISSUE TITLE
Permanently disconnect from a node

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#336]
As discussed in https://github.com/ExchangeUnion/xud/issues/305#issuecomment-411606536 we should permanently disconnect from a peer on manual `disconnect` via gRPC/CLI.

Easiest proposed solution: add to `bannedHosts`.

**Comment by sangaman** [comment]
Yeah I'd use the banning functionality unless we have a reason not to.

**Comment by kilrau** [comment]
Merge 'disconnect' into ban, which permanently disconnect, sets host on ban list and doesn't allow incoming connections from this host

**Comment by sangaman** [comment]
We would be renaming `disconnect` to `ban`, right?

**Comment by sangaman** [comment]
Before we implement this we should have banning done right first. We should add a `hostReputationEvents` table as discussed in #185. We should add a row to that table with the hostId that's being banned, and the event (in this case a manual banning). I think it would be nice to have an internal enum for all the reputation events there can be, and in the db we only store the number value of the enum. This event could be `MANUAL_BAN`. We'd also want to update the `hosts` table to set the `banned` field to true.

Thoughts @moshababo?

**Comment by michael1011** [comment]
Yeah I agree on renaming it to `ban` and on not storing the ban score itself in the database. We should move this issue to alpha.2 because so is #185 and we don't necessarily need this right now.

**Comment by kilrau** [comment]
Agreed!

**Comment by michael1011** [comment]
Depends on #457
